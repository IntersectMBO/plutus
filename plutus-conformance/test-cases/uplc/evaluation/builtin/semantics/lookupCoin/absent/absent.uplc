(program 1.0.0
  [ (builtin lookupCoin)
    (con bytestring #aa)
    (con bytestring #bb)
    (con value [(#aa, [(#aa, 100)]), (#bb, [(#aa, 1)])])
  ]
)
