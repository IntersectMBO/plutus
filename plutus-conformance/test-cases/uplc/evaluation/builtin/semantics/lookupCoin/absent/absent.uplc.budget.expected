({cpu: 338744
| mem: 801})